Automattic/woocommerce-payments
# Update @wordpress/components to the latest version

Fixes #717

#### Changes proposed in this Pull Request

* Update `@wordpress/components` dependency
* Fixes all UI quirks after update
* Resolves instant-deposits UI TODOs introduced by #1437

#### Testing instructions

* Checks should be green.
* Update dependencies with `npm install`.
* Launch site and check the UI.

-------------------
